spec/models/form/lettings/questions/homeless_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
```

Comment:
Right, I've added a test file here - though I am still a little concerned that all I've achieved is to make this file more difficult to change!

Reply:
Basically I agree with you that there's not much value added with our current approach to testing questions and pages, I think it's worth visiting more generally - maybe one for team time?

Reply:
Yeah, I don't disagree with any of that!